pkg/systray/runtime/runner.go

```diff
@@ -70,4 +73,6 @@ func (r *SystrayUsersProcessesRunner) Interrupt(err error) {
 			)
 		}
 	}
+
+	r.procsWg.Wait()
```

Comment:
If I understand this correctly, what happens if there's a systray process wedged hard? This would block launcher from exiting?

Reply:
added a 5 second time out for this